akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -278,6 +278,8 @@ object HttpEntity {
   implicit def apply(data: ByteString): HttpEntity.Strict = apply(ContentTypes.`application/octet-stream`, data)
   def apply(contentType: ContentType.NonBinary, string: String): HttpEntity.Strict =
     if (string.isEmpty) empty(contentType) else apply(contentType, ByteString(string.getBytes(contentType.charset.nioCharset)))
+  def apply(contentType: ContentType.WithFixedCharset, string: String): HttpEntity.Strict =
+    if (string.isEmpty) empty(contentType) else apply(contentType, ByteString(string.getBytes()))
```

Comment:
For the `apply(ContentType, ByteString)` part, which one do you think is better?;
`apply(contentType, ByteString(string.getBytes())`
OR
`apply(contentType, ByteString(string.getBytes(contentType.charset.nioCharset)))`

Reply:
The second.

```suggestion
    if (string.isEmpty) empty(contentType) else apply(contentType, ByteString(string.getBytes(contentType.charset.nioCharset))))
```